app/models/users_languages.php

```diff
@@ -190,7 +191,15 @@ public function getNumberOfUsersForEachLanguage()
             array(
                 'fields' => array('language_code', 'COUNT(*) as total'),
                 'group' => 'language_code',
-                'order' => 'total DESC'
+                'conditions' => array(
+                    'User.group_id != 6'
+                ),
+                'order' => 'total DESC',
+                'contain' => array(
+                    'User' => array(
```

Comment:
Here you can just do: `'contain' => array('User')`. There's no need to add the an empty `fields` array.